gulp-utils.js

```diff
@@ -3,7 +3,7 @@ function generateBindings(inputFile, outputFile, callback) {
   const asc = getAsc();
   asc.main([
     inputFile,
-    "--baseDir", "assembly",
+    "--baseDir", process.cwd(),
```

Comment:
it's likely not a thing in browser (when running in studio)

Reply:
You could just mock it, at least that's what [BrowserFS](https://github.com/jvilk/browserfs) does.